### PR TITLE
Admin tweaks and fixes

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/stylesheets/admin/base.scss [hunk 1]
```diff
@@ -1195,3 +1196,8 @@ table .ellipsis {
     }
   }
 }
+
+.search-form h2 .btn-link {
+  padding-left: 0;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered margin-right, padding-left

File: app/assets/stylesheets/admin/page-settings.scss [hunk 2]
```diff
@@ -57,16 +56,68 @@
...
+
+      .btn {
+        @include screen-sm-max {
+          display: block;
+          width: 100%;
+          padding-left: 0;
+          padding-right: 0;
+        }
+
+        &:first-child {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-settings.scss]: Nesting should be no greater than 4, but was 5

File: app/assets/stylesheets/admin/page-settings.scss [hunk 3]
```diff
@@ -57,16 +56,68 @@
...
 
     .control-action {
+      position: relative;
       margin-top: 1.7em;
+
+      .btn {
+        @include screen-sm-max {
+          display: block;
+          width: 100%;
+          padding-left: 0;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-settings.scss]: Properties should be ordered display, width, padding-right, padding-left

**Comment by RuslanHamidullin** [comment]
Looks good :+1:

**Pull request merged**